modules/nvidia-gpu-admin-dashboard-using.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
+
+== Viewing the cluster GPU overview
+
+You can view the status of your cluster GPUs in the Overview page by selecting
+Overview in the Home section.
+
+The Overview page provides information about the cluster GPUs, including:
+
+* Details about the GPU providers
+* Status of the GPUs
+* Cluster utilization of the GPUs
+
+== Viewing the GPUs dashboard
+
+You can view the NVIDIA GPU administration dashboard by selecting GPUs
+in the Compute section of the OpenShift Console.
+
+Charts on the GPUs dashboard include:
+
+* GPU utilization
+* Memory utilization
+* Encoder utilization
+* Decoder utilization
+* Power consumption
+* GPU temperature
+* GPU clock speed
+* Memory clock speed
+
+Each chart type explains:
+
+* What the chart represents
+* Why the chart is important
+* Expected and typical results
+* How to potentially mitigate bad results
+
+
+== Viewing the GPU Metrics
+
+You can view the metrics for the GPUs by selecting the metric at the bottom of
+each GPU to view the Metrics page.
+
+On the Metrics page, you can:
```

Comment:
Do we need more descriptive text here?

Reply:
Do we really want to tell customers where to find all the metrics? Isn't it out of scope of the GPU dashboard?

Reply:
@fabiendupont The Metrics page is highly accessible to the user, so I think it's good information. However, let me know if you feel strongly about removing it.